frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -1,23 +1,580 @@
+        {data.case.otherFeeToWaive && (
+          <>
+            <TextInput label="Explain" dataKey="case.otherFeeToWaive.name" />
+            <TextInput label="Amount" dataKey="case.otherFeeToWaive.amount" />
+          </>
+        )}
+      </Section>
+
+      <Section name="4. Employment">
+        <CheckBox
+          dataKey="case.employed"
+          label="I am employed"
+          name="employmentStatuses"
+        />
+        {data.case.employed && (
+          <>
+            <small className="web-form-input">(Choose all that apply):</small>
+            <CheckBox
+              dataKey="case.hourlyEmployee"
+              label="An hourly employee"
+              name="employmentType"
+            />
+            <small className="web-form-input">(Form W-2)</small>
+            <CheckBox
+              dataKey="case.salaryEmployee"
+              label="A salaried employee"
+              name="employmentType"
+            />
+            <small className="web-form-input">(Form W-2)</small>
+            <CheckBox
+              dataKey="case.selfEmployed"
+              label="Self-employed"
+              name="employmentType"
+            />
+            <small className="web-form-input">
+              (Form 1099, Form K-1, Schedule C, etc.)
+            </small>
+            <CheckBox
+              dataKey="case.otherEmployment"
+              label="other"
+              name="employmentType"
+            />
+            {data.case.otherEmployment && (
+              <TextInput dataKey="case.otherEmployeeExplain" label="Explain" />
+            )}
+          </>
+        )}
+        <CheckBox
+          dataKey="case.unemployed"
+          label="I am unemployed"
+          name="employmentStatuses"
+        />
+        {data.case.unemployed && (
+          <TextArea dataKey="case.unemployedExplanation" label="Explain" />
+        )}
+      </Section>
+
+      <Section name="5. Dependents">
+        <p className="web-form-input">
+          The following people depend on me for support
+        </p>
+        <small className="web-form-input">
+          (Count Spouse, children or other dependents in your house hold.)
+        </small>
+        <TextInput label="Number of adults" dataKey="case.adultDependents" />
+        <TextInput
+          label="Number of children under 18"
+          dataKey="case.childrenDependents"
+        />
+      </Section>
+
+      <Section name="6. Gross Monthly Income">
+        <CheckBox
+          label="I have the following monthly income before tax deductions:"
+          name="grossMonthlyIncome"
+          dataKey="person.grossMonthlyIncome"
+        />
+        <small className="web-form-input">
+          (For income that changes from month to month, calculate the annual
+          total and divide by 12 months to list a monthly average.)
+        </small>
+        {data.person.grossMonthlyIncome && (
+          <>
+            <TextInput label="Work" dataKey="person.income.work" />
```

Comment:
to match the other income data keys, maybe call this `person.income.workIncome`?